src/engraving/rw/compat/compatutils.cpp

```diff
@@ -103,6 +106,10 @@ void CompatUtils::doCompatibilityConversions(MasterScore* masterScore)
     if (masterScore->mscVersion() < 440) {
         mapHeaderFooterStyles(masterScore);
     }
+
+    if (masterScore->mscVersion() < 450) {
```

Comment:
Have we already bumped the file version number? If not, let's do it soon, I'll need this too :)

Reply:
Not yet!